video/out/wayland_common.c

```diff
@@ -2106,8 +2110,6 @@ void vo_wayland_uninit(struct vo *vo)
     wl_list_for_each_safe(output, tmp, &wl->output_list, link)
         remove_output(output);
 
-    talloc_free(wl->drm_formats);
```

Comment:
`MPV_LEAK_REPORT=1` tells me that there's actually a leak without this here. Kind of weird because in theory it shouldn't be needed.